configs/pytorch.json

```diff
@@ -1,8 +1,14 @@
 {
   "index_name": "pytorch",
   "start_urls": [
+    "https://pytorch.org",
+    "https://pytorch.org/index.html",
     "https://pytorch.org/get-started/",
-    "https://pytorch.org/get-started/locally/"
+    "https://pytorch.org/get-started/locally/",
```

Comment:
This page is already crawled

Reply:
That page was already there, I just added a comma `,`.